src/Setup/Model.php

```diff
@@ -56,6 +56,10 @@
  *   Keys should be prefixed based on which plugin manages the field.
  *   Values must only be scalars (bool/int/string) and arrays.
  *   Ex: ['opt-in.version-check' => TRUE].
+ * @property array $tables
+ *   List of CiviCRM tables extracted from xml schema files.
+ *   Values must only be array.
+ *   Ex: ['civicrm_contact', 'civicrm_group'].
```

Comment:
I think putting some schema metadata in `Model` makes in terms of dataflows, although I wouldn't any downstream installer code (like `cv core:install` or D8 `civicrm.install`) to manipulate those fields.

FWIW, if we wanted this to be private, it could go in something like `$model->extras['schema.tables']`